src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -169,21 +161,13 @@ private static bool SignatureEquals(IMethodSymbol? method, IMethodSymbol? baseMe
                 }
             }
 
+            // TODO: GenAPI does not currently preserve __arglist as a parameter.
```

Comment:
Can you make sure to file an issue for this?

Reply:
Filed https://github.com/dotnet/sdk/issues/31196